phalcon/mvc/model/query.zep

```diff
@@ -178,6 +180,9 @@ class Query implements QueryInterface, InjectionAwareInterface
 		} else {
 			let this->_enableImplicitJoins = globals_get("orm.enable_implicit_joins");
 		}
+
+		let this->_nestingLevel = -1;
+		let this->_sqlColumnAliases = [];
```

Comment:
Same here, set `[]` as the default property on the class.

Reply:
Good point :)